apps/ltn/src/browse.rs

```diff
@@ -106,6 +108,9 @@ impl State<App> for BrowseNeighborhoods {
                     });
                     return Transition::Replace(BrowseNeighborhoods::new_state(ctx, app));
                 }
+                "hide panel" | "show panel" => {
```

Comment:
This is a verbose implementation; 5 different places in the code need to match these actions and delegate. This is a symptom of a larger problem with getting different "components" sharing one physical panel to compose nicely. There's already some attempt to delegate a panel outcome to multiple places for `AltProposals`.

An old idea I'm considering reviving: bundling a `Panel` with something that can just register "callbacks" or handlers, mutably in a builder style. I'll experiment with that...

And the even more dramatic implementation would make hideable panels a first-class widgetry concept, almost similar to how scrollbars work. Panels would consume the hide/show event internally, and application-level code would never even see anything. Panels would always be constructed by apps as if they're visible, but internally have some state to track hidden/visible. This gets complicated with states that constantly recreate panels -- we probably need to do `panel.restore` in those cases. It also makes minimizedness a per-panel property, so if you
1) browse neighborhoods
2) hide the panel
3) click a neighborhood
the left panel would be visible in the viewer that shows up, unlike the video in this PR